dotCMS/src/main/java/com/dotcms/rest/MapToContentletPopulator.java

```diff
@@ -287,6 +289,9 @@ private void processSeveralHostsOrFolders(final Contentlet contentlet,
     public List<Category> getCategories (final Contentlet contentlet, final User user,
                                          final boolean respectFrontendRoles) throws DotDataException, DotSecurityException {
 
+        DotPreconditions.checkNotNull(contentlet, IllegalArgumentException.class, "Invalid Content Type");
```

Comment:
should not be another message since it's the same than below, maybe like **Invalid Contentlet**??

Reply:
will change